core/src/main/java/org/mskcc/cbio/portal/model/AlleleSpecificCopyNumber.java

```diff
@@ -32,9 +32,13 @@
 
 package org.mskcc.cbio.portal.model;
 
+import org.mskcc.cbio.maf.ValueTypeUtil;
```

Comment:
Do we need this? The static import statement below should suffice, right?

```Java
import static org.mskcc.cbio.maf.ValueTypeUtil.*;
```

Reply:
Import were updated.